src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
+function validateTriangle(a: number, b: number, c: number): void {
+  if (
+    a <= 0
+    || b <= 0
+    || c <= 0
+  ) {
```

Comment:
pretty clear enough this way, I believe. Or you can create a variable for that - would be even better
```suggestion
  if (a <= 0 || b <= 0 || c <= 0) {
```